source/uwp/Renderer/lib/AdaptiveActionSetRenderer.cpp

```diff
@@ -2,44 +2,64 @@
 // Licensed under the MIT License.
 #include "pch.h"
 
-#include "AdaptiveActionSet.h"
 #include "AdaptiveActionSetRenderer.h"
+
+#include "ActionHelpers.h"
+#include "AdaptiveActionSet.h"
 #include "AdaptiveElementParserRegistration.h"
-#include "enums.h"
-#include "Util.h"
-#include "XamlBuilder.h"
+#include "AdaptiveRenderArgs.h"
 
 using namespace Microsoft::WRL;
 using namespace Microsoft::WRL::Wrappers;
 using namespace ABI::AdaptiveNamespace;
 using namespace ABI::Windows::Foundation;
+using namespace ABI::Windows::Foundation::Collections;
+using namespace ABI::Windows::UI::Xaml;
+using namespace ABI::Windows::UI::Xaml::Controls;
+using namespace ABI::Windows::UI::Xaml::Controls::Primitives;
 
 namespace AdaptiveNamespace
 {
-    HRESULT AdaptiveActionSetRenderer::RuntimeClassInitialize() noexcept try
-    {
-        return S_OK;
-    }
-    CATCH_RETURN;
-
-    HRESULT AdaptiveActionSetRenderer::Render(_In_ IAdaptiveCardElement* cardElement,
-                                              _In_ IAdaptiveRenderContext* renderContext,
-                                              _In_ IAdaptiveRenderArgs* renderArgs,
-                                              _COM_Outptr_ ABI::Windows::UI::Xaml::IUIElement** result) noexcept try
-    {
-        return XamlBuilder::BuildActionSet(cardElement, renderContext, renderArgs, result);
-    }
-    CATCH_RETURN;
+    HRESULT AdaptiveActionSetRenderer::RuntimeClassInitialize() noexcept { return S_OK; }
 
     HRESULT AdaptiveActionSetRenderer::FromJson(
         _In_ ABI::Windows::Data::Json::IJsonObject* jsonObject,
         _In_ ABI::AdaptiveNamespace::IAdaptiveElementParserRegistration* elementParserRegistration,
         _In_ ABI::AdaptiveNamespace::IAdaptiveActionParserRegistration* actionParserRegistration,
         _In_ ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveNamespace::AdaptiveWarning*>* adaptiveWarnings,
-        _COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveCardElement** element) noexcept try
+        _COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveCardElement** element) noexcept
+    try
     {
         return AdaptiveNamespace::FromJson<AdaptiveNamespace::AdaptiveActionSet, AdaptiveSharedNamespace::ActionSet, AdaptiveSharedNamespace::ActionSetParser>(
             jsonObject, elementParserRegistration, actionParserRegistration, adaptiveWarnings, element);
     }
     CATCH_RETURN;
+
+    HRESULT AdaptiveActionSetRenderer::Render(_In_ IAdaptiveCardElement* adaptiveCardElement,
+                                              _In_ IAdaptiveRenderContext* renderContext,
+                                              _In_ IAdaptiveRenderArgs* renderArgs,
+                                              _COM_Outptr_ IUIElement** actionSetControl) noexcept
+    try
```

Comment:
>try [](start = 4, length = 3)

I assume this is automated, but any idea why sometimes these trys are on their own line and sometimes they're on the line with the noexcept?

Reply:
I was manually fixing some of these up, as I thought it was a quirk in my setup. Probably not, though, so I'll re-format everything :)

---
In reply to: [327215358](https://github.com/microsoft/AdaptiveCards/pull/3457#discussion_r327215358) [](ancestors = 327215358)